app/src/main/java/com/codingblocks/cbonlineapp/auth/onboarding/SignInFragment.kt

```diff
@@ -19,7 +19,7 @@ import kotlinx.android.synthetic.main.fragment_sign_in.*
 
 class SignInFragment : Fragment() {
 
-    lateinit var type: String
+    var type: String = ""
```

Comment:
App was crashing due to uninitialized value for this. Also default value was empty string only. So replaced with this